tool/dependencies/known-dependencies.txt

```diff
@@ -140,8 +147,10 @@ flink-table-common-1.12.2.jar
 flink-table-planner-blink_2.11-1.12.2.jar
 flink-table-runtime-blink_2.11-1.12.2.jar
 flink-yarn_2.11-1.12.2.jar
+flyway-core-6.4.4.jar
```

Comment:
the dependency jar should deal with guide of 
https://linkis.apache.org/zh-CN/community/development_specification/license